src/components/plots/barPlot.ts

```diff
@@ -214,6 +213,14 @@ export module Plot {
       };
     }
 
+    protected _isVisibleOnPlot(datum: any, pixelPoint: Point, selection: D3.Selection): boolean {
+      var chartXExtent = { min: 0, max: this.width() };
+      var chartYExtent = { min: 0, max: this.height() };
```

Comment:
"xRange" and "yRange"? We use "extent" elsewhere to refer to a pair of min-max values in data-space.